scripts/restore

```diff
@@ -106,5 +107,12 @@ fi
 #=================================================
 # RELOAD NGINX
 #=================================================
-systemctl restart $app
+
 systemctl reload nginx
+
+#=================================================
+# START SERVICE
+#=================================================
+
+ynh_systemd_action --action=start --service_name=$app --log_path="systemd" 
```

Comment:
No `--line_match` here ? Is that voluntary ? :stuck_out_tongue:

Reply:
when the system start after restore: here are the log in journalctl
![wekan](https://user-images.githubusercontent.com/30271971/52516287-7ed96080-2c28-11e9-8e5d-2516eecc5d46.PNG)
In my opinion, nothing helpful to define that wekan is fully started.
But I can add: --line_match="Started Wekan"
Let me know